src/App.js

```diff
@@ -33,21 +34,27 @@ function App() {
         <Route
           exact
           path="/teams"
+          user={currentUser}
           render={(routeProps) => <Teams {...routeProps} user={currentUser} />}
         />
-        <Route
-          exact
-          path="/teams/new"
-          render={(routeProps) => <AddTeam {...routeProps} user={currentUser} />}
-        />
+        <ProtectedRoute currentUser={currentUser} exact path="/teams/new">
```

Comment:
You shouldn't need the Route here. It's already in the ProtectedRoute component.